lessons/Data-Science/ai-decision-trees/langs/en-us/pages/build-and-test.adoc

```diff
@@ -0,0 +1,80 @@
+= Building and Testing a Decision Tree
+
+On @printable-exercise{level-1.adoc}, we decided that the decision mode at the end of the "teens" branch should be "interest in game". Let's create the rest of the tree and test it out. You can refer to the training dataset on @handout{decision-tree-data.adoc}.
+
+== Build and Understand the Tree
+
+@vspace{1ex}
+
+Fill in the remainder of the tree using the algorithm you have learned (left) before filling in the blanks (right).
+
+@vspace{1ex}
+
+[cols="1,1", stripes="none"]
+|===
+
+|
+@ifnotsoln{@center{@image{../images/incomplete-tree.png, 350}}}
+
+@ifsoln{@center{@image{../images/final-tree.png, 350}}}
+
+|
+
+@n The root node of this tree is @fitb{10em}{@ifsoln{age}}.
```

Comment:
Can we let the underlines take up the whole space? The jagged line endings are a little jarring